File: src/test/java/Day11Test.java
import org.junit.Test;

import static org.junit.Assert.*;

public class Day11Test {
    @Test
    public void testPart1() throws Exception {
        assertEquals(3, Day11.part1("ne,ne,ne"));
        assertEquals(0, Day11.part1("ne,ne,sw,sw"));
        assertEquals(2, Day11.part1("ne,ne,s,s"));
        assertEquals(3, Day11.part1("se,sw,se,sw,sw"));
    }

    @Test
    public void testPart2() throws Exception {
        assertEquals(0, Day11.part2("ne,ne,sw,sw"));
    }
}

File: src/test/java/aoc2018/Day11Test.kt
package aoc2018

import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class Day11Test {

    @Test
    fun part1() {
        val input = " /n ".split("\n")
        assertThat(Day11.part1(input)).isEqualTo(1)
    }

    @Test
    fun part2() {
        val input = " /n ".split("\n")
        assertThat(Day11.part2(input)).isEqualTo(2)
    }
}
